recruiter/models.py

```diff
@@ -35,3 +35,13 @@ class Recruiter(models.Model):
 
     def __str__(self):
         return f'{self.name}'
+
+    @classmethod
+    def is_recruiter(cls, auth_user_id):
+        """
+        For checking if a currently logged in user is a Recruiter
+        """
+        user = cls.objects.filter(user_id=auth_user_id)
+        if user:
+            return True
+        return False
```

Comment:
I don't understand this check - if it's a user than it's necessarily a recruiter?

Reply:
@guy9050 
I'm checking if there exists a Recruiter with a PK that matches the given auth user id.
If a Recruiter that matches the search is found it will be stored in the "user" QuerySet.
The statement `if user` checks if the QuerySet is empty, if we go into the if statement it means that it isn't empty, meaning there's a recruiter that matched the search. Otherwise, the query set is empty and we return false.

You can think of the `if user` statement as `if (! user.isEmpty())`

Reply:
I added comments to the functions, hopefully it's clear now